webapp/components/ContributionForm/ContributionForm.spec.js

```diff
@@ -275,11 +299,20 @@ describe('ContributionForm.vue', () => {
           postTitleInput.setValue(postTitle)
           await wrapper.vm.updateEditorContent(postContent)
           categoryIds = ['cat12']
-          wrapper.find(CategoriesSelect).vm.$emit('updateCategories', categoryIds)
+          wrapper.find(CategoriesSelect).setData({ categories })
+          wrapper
+            .findAll('li')
+            .at(1)
+            .trigger('click') // language
+          await wrapper
+            .find(CategoriesSelect)
+            .findAll('button')
+            .at(1)
```

Comment:
all of these, I think...
I like in enzyme where you have `filterWhere`, but you could just use `filter` from javascript as well... but the idea is that if someone adds a button the location will change, and all the tests will need to be updated, no?